portfolio/src/main/java/com/google/sps/servlets/LoginStatusServlet.java

```diff
@@ -0,0 +1,42 @@
+package com.google.sps.servlets;
+
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import com.google.gson.Gson;
+import com.google.appengine.api.users.UserService;
+import com.google.appengine.api.users.UserServiceFactory;
+
+import com.google.sps.data.LoginStatus;
+
+
+@WebServlet("/loginStatus")
+public class LoginStatusServlet extends HttpServlet {
+
+    private static final UserService userService = UserServiceFactory.getUserService();
+    private static final Gson gson = new Gson();
+
+    @Override
+    public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException  {
+        response.setContentType("application/json");
+        
+        LoginStatus loginStatus;
+
+        if (userService.isUserLoggedIn()) {
+            String userEmail = userService.getCurrentUser().getEmail();
+            String goToHome = "/comments.html";
```

Comment:
You could move this outside of the if check, so you don't have to write the same line in the else block.

Reply:
Alternatively, you can have this be a Java constant